sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -525,6 +525,11 @@ public static TableRow toTableRow(Row row) {
         return toTableRow((Row) fieldValue);
 
       case DATETIME:
+        if (fieldValue instanceof Long) {
+          return Instant.ofEpochMilli((long) fieldValue)
+              .toDateTime(DateTimeZone.UTC)
+              .toString(BIGQUERY_TIMESTAMP_PRINTER);
+        }
```

Comment:
Why is this necessary?

Reply:
This validation comes from [this](https://github.com/apache/beam/pull/10369/commits/79be403dc3f784a241a4129d987dc5a126f34568) commit in the previous PR, I'm struggling to find why it was added, as I don't see any change that gives the possibility of the DATETIME being a Long instead of Instant. Should this be removed?

Reply:
Hm yes lets remove it if doing so doesn't break any tests.